pkg/modules/application_mobility.go

```diff
@@ -625,18 +622,6 @@ func getVelero(op utils.OperatorConfig, cr csmv1.ContainerStorageModule) (string
 				veleroImgPullPolicy = string(component.ImagePullPolicy)
 			}
 			for _, env := range component.Envs {
-				if strings.Contains(BackupStorageLocation, env.Name) {
```

Comment:
It looks like we're converting some env variables -> config variables. but these changes are not made in the samples file?